e2e/drone_setup.sh

```diff
@@ -14,6 +14,15 @@ do
   echo "Apply failed... trying again in 5s"
   sleep 5
 done
-terraform output -json > tf.json
+terraform output -json > tf.json.tmp
+
+jq 'del(.pharos_hosts.value.worker_up)' tf.json.tmp > tf.json
```

Comment:
Should this be within `if [ "$1" = "ubuntu-18-04-x64" ]; then` ?

Reply:
The `worker_up` section needs to be removed always or the generated config isn't valid.

Reply:
Maybe it should not be under `pharos_hosts` then?